src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -414,6 +413,16 @@ private void RaiseSessionSpansUpdated(ImmutableArray<InlineRenameLocation> locat
         {
             AssertIsForeground();
             SetReferenceLocations(locations);
+
+            // If workspace would ignore changes to unchangeable documents, we filter out references in 
+            // them only to avoid displaying them in UI, i.e. everything else in inline rename is still
+            // oblivious to unchangeable document.
```

Comment:
It might be worth adding a comment here that it's OK that SetReferenceLocations was called with the original data _first_, and the unchageable documents can't be opened so the _openTextBuffer loop there won't matter. This matches what we discussed in person but somebody who *wasn't* party to that discussion will think this is a bug.